### PR TITLE
prepare release

### PR DESCRIPTION
**Pull request by goessebr**

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -49,8 +48,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'skosprovider_getty'
-copyright = u'2014-2020, Flanders Heritage Agency'
+project = 'skosprovider_getty'
+copyright = '2014-2020, Flanders Heritage Agency'
```
Comment (koenedaele) [inline docs/source/conf.py:52]: 2014-2021

File: docs/source/conf.py [hunk 2]
```diff
@@ -59,7 +58,7 @@
 # The short X.Y version.
 version = '0.5'
 # The full version, including alpha/beta/rc tags.
-release = '0.5.0'
+release = '0.5.1'
```
Comment (koenedaele) [inline docs/source/conf.py:61]: 1.0.0

File: .travis.yml [hunk 3]
```diff
@@ -1,10 +1,9 @@
 sudo: false
 language: python
 python:
-- 2.7
 - 3.6
-- 3.7
```
Comment (koenedaele) [inline .travis.yml:6]: Why do we drop 3.7, but maintain 3.6, 3.8 and 3.9?

**Review by koenedaele — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 3 above)
Comment (goessebr) [reply .travis.yml:6]: Based on https://github.com/OnroerendErfgoed/skosprovider_getty/pull/87#pullrequestreview-815061712

Comment (koenedaele) [reply .travis.yml:6]: That explains why we kept compatibility with 3.6. This is an OSS library that should follow it's own logic. If we maintain 3.6 compatibility for longer, then we should also maintain 3.7 compatibility. This takes up a bit more travis resources, but especially for a small library such as this, it's minimal. For applications such as atramhasis (and especially internal stuff) it makes more sense to drop versions to reduce the load on travis.

Comment (goessebr) [reply .travis.yml:6]: I agree

**Pull request merged**